client/src/Login/Login.js

```diff
@@ -7,25 +7,54 @@ import Button from 'react-bootstrap/Button';
 import CardGroup from 'react-bootstrap/CardGroup';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
-import {LoginConstants} from "../__helpers__/Constants";
-
+import { LoginConstants, ToastConstants } from "../__helpers__/Constants";
+import Toast from '../Toast/Toast';
 
 
 // FIXME: Review formating of extra columns
 export default function Login() {
     const [validated, setValidated] = useState(false);
+    const [isHidden, setHidden] = useState(true);
+    const [color, setColor] = useState(null);
+    const [message, setMessage] = useState(null);
 
     const handleSubmit = (e) => {
         const form = e.currentTarget;
+
         if (form.checkValidity() === false) {
             e.preventDefault();
             e.stopPropagation();
         }
-
         setValidated(true);
     }
 
+    const handleErrors = (e) => {
+        let username = document.getElementById('email').value;
```

Comment:
There's a better option than accessing DOM elements directly.  While not forbidden in react, it isn't the intent of the framework.

Reply:
Done